transcom/mymove
# B-18418 Fixed visibility of delete shipment on moves that are already approved

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A882683&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

Removed the delete shipment button from the edit move page for moves that have been approved

### How to test

1. Access the office app
2. Login as a TOO
3. Edit a PPM/HHG move that hasn't been approved
4. Delete Shipment should not be visible

### Frontend
